RealmBrowser/Views/RLMBasicTableCellView.m

```diff
@@ -32,9 +32,15 @@ - (instancetype)initWithFrame:(NSRect)frameRect
     textField.translatesAutoresizingMaskIntoConstraints = NO;
     textField.bordered = NO;
     textField.drawsBackground = NO;
-    textField.usesSingleLineMode = YES;
-    textField.lineBreakMode = NSLineBreakByTruncatingTail;
     
+    if ([textField respondsToSelector:@selector(setUsesSingleLineMode:)]) {
+        textField.usesSingleLineMode = YES;
```

Comment:
Every instance of `NSControl`, of which `NSTextField` is a subclass, has a corresponding cell. I'm proposing that we set `usesSingleLineMode` and `lineBreakMode` on the cell rather than on the text field as those properties have existed on the cell class for longer than they have on `NSTextField` itself.